include/solarcharger/victron/Stats.h

```diff
@@ -25,6 +25,7 @@ class Stats : public ::SolarChargers::Stats {
     void mqttPublishSensors(const boolean forcePublish) const final;
 
     void update(const String serial, const std::optional<VeDirectMpptController::data_t> mpptData, uint32_t lastUpdate) const;
+    void clean();
```

Comment:
```suggestion
    void cleanStaleEntries();
```

`clean` sounds like we remove all the stats

Reply:
Done.